### DISCUSSION TITLE
installing entreprise edition on a different port than 443

### DISCUSSION THREAD

**Discussion by nsunder72** [bionic-gpt/bionic-gpt#373] [Q&A]
Hello, first the installation process on 443 works fine but i need to install it behind a webserver, so i try to install using
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
  kubeadmConfigPatches:
  - |
    kind: InitConfiguration
    nodeRegistration:
      kubeletExtraArgs:
        node-labels: "ingress-ready=true"
  extraPortMappings:
  - containerPort: 443
    hostPort: **8989**

but now when i try to connect to it with https://localhost:8989 i get
Your connection is not private NET::ERR_CERT_AUTHORITY_INVALID

please help i tried to install a certificate on localhost but it doesn't change anything
and even if i bypass the warning, the browser jumps to 
https://localhost/oidc/realms/bionic-gpt/protocol/openid-connect/auth?approval_prompt=force&client_id=bionic-gpt&redirect_uri=https%3A%2F%2Flocalhost%2Foauth2%2Fcallback&response_type=code&scope=openid+email+profile&state=cK4HN2Y_LvXklnHZqclS4LYHVgcMBDLQ-MGSzeAHRIk%3A%2F
and loos the port parameter :8989 failing to connect anyway>>>

**Comment by ianpurton** [comment]
When you say install behind a web server what does that mean?

Did you try the nginx setup?

**Reply by nsunder72** [reply]
it means the same machine has a front webserver on 443 so it's taken
